test/alu_8bit_sub_test.c

```diff
@@ -103,8 +102,8 @@ Test(sub_a_n, sub_a_c_borrow, .init = cpu_mmu_setup, .fini = cpu_teardown) {
 
 Test(sub_a_n, sub_a_d_res_only, .init = cpu_mmu_setup, .fini = cpu_teardown) {
     const uint8_t opcode = 0x92;
-    const uint8_t a = rand() % (0xFFFF);
-    const uint8_t d = rand() % (0xFFFF);
+    const uint8_t a = rand() % (0xFF);
```

Comment:
I think it would be a good idea to remove the `% (0xFF)`, etc.
See here: https://godbolt.org/z/ec6W7Mfhh